Functions.Templates/Templates/BlobTrigger-Python/function.json

```diff
@@ -0,0 +1,13 @@
+{
+    "scriptFile": "__init__.py",
+    "disabled": false,
+    "bindings": [
+        {
+            "name": "my_blob",

```

Comment:
The functions host yells if you give it an `_` in the `name`. Can you change this to `myblob`, along with any other instances in the code?